src/version/version.rs

```diff
@@ -116,7 +139,10 @@ pub fn background_check_for_updates() -> mpsc::Receiver<Version> {
 
     let _detached_thread = thread::spawn(move || match check_wrangler_versions() {
         Ok(wrangler_versions) => {
-            if wrangler_versions.current != wrangler_versions.latest {
+            // If the wrangler version has not been checked within the last day and the versions
+            // are different, print out an update message
+            if !wrangler_versions.checked && (wrangler_versions.current != wrangler_versions.latest)
```

Comment:
nit: this might be easier to grok as a method `outdated(&self)` on the `WranglerVersion` struct.